web/src/types.ts

```diff
@@ -103,6 +103,13 @@ export type Snapshot = {
   SnapshotLink: string
   showSnapshotModal: boolean
   path?: string
+  snapshotHighlight?: SnapshotHiglight | null
+}
+
+export type SnapshotHiglight = {
+  beginningLogID: string
```

Comment:
we should probably handle offset as well? traditionally selection state is represented like
startLine
startOffset
endLine
endOffset